### DISCUSSION TITLE
Comparison image added fingers

### DISCUSSION THREAD

**Discussion by creeduk** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#104] [General]
I noted on this one - https://imgsli.com/MTY2MTcx that there are some extra undesired fingers which I now is a hazard with SD. Any best setting for that image. Looks like faces, lips, eyes come out much better than the other scaler, but details like hair and sticking to original details of fingers etc. is handled better by ultimate so a Photoshop merge would give the ultimate result. Or some inpainting. Still interested to see if that image could be done with the great face but not adding the extra fingers in 1 pass?

**Comment by pkuliyi2015** [comment]
The extra fingers appears in my demo because the original image has mess fingers.
If your original image is fine, then the upscaled image is not likely to add extra fingers when you set renoise strength = 0.
